Zend/zend_globals.h

```diff
@@ -64,6 +64,7 @@ END_EXTERN_C()
 
 typedef struct _zend_declarables {
 	zval ticks;
+	zend_bool strict_types;
```

Comment:
This seems to be a remnant of an earlier refactor and can be removed now. I see no other reference to declarables in the patch. I simply forgot to remove this line.